StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoBodyView.swift

```diff
@@ -126,6 +135,81 @@ private class AutoResizableImageView: UIImageView {
     }
 }
 
+// MARK: - Bullets
+
+private func BulletsBodyEntryView(
```

Comment:
[Stripe.js component](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/0ca23d8dda58873c9c793ae8ccdf980e08cf5f3b/src/linkedAccounts/inner/components/GenericInfoScreen/index.tsx#L189-L213)

I kept some of the iOS values for text sizing a _little bit_ different (but not much). This is because other parts of the app utilize `BulletPointLabelView` and I don't want to change an already established design.